### PR TITLE
Fix relative pathname lookup while installing a local .gem file

### PR DESCRIPTION
**Pull request by purbon**
Fix the Pathname lookup for installing new .gem files.

### CODE REVIEW

**Comment by purbon** [comment]
@suyograo @ph review on here?

**Comment by ph** [comment]
@purbon it was failing?  I think we require "pathname" globably ?

**Comment by purbon** [comment]
I tested master and 1.5 and not.

**Comment by purbon** [comment]
and btw, we should not require pathname globally :-)

File: lib/logstash/pluginmanager/command.rb [hunk 1]
```diff
@@ -32,6 +32,7 @@ def remove_unused_locally_installed_gems!
   end
 
   def relative_path(path)
-    Pathname.new(path).relative_path_from(Pathname.new(LogStash::Environment::LOGSTASH_HOME)).to_s
+    require 'pathname'
```
Comment (ph) [inline lib/logstash/pluginmanager/command.rb]: double quotes and merge.

**Comment by ph** [comment]
:shipit: lgtm

**Pull request closed without merging**